test/extended/testdata/oauthserver/oauth-pod.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test-oauth-server
+  name: test-oauth-server
+spec:
+  containers:
+  - command:
+    - oauth-server
+    - osinserver
+    - --config=/var/config/system/configmaps/oauth-config/oauth.conf
+    - --v=100
+    image: quay.io/openshift/origin-oauth-server:latest
+    imagePullPolicy: IfNotPresent
+    name: oauth-server
+    ports:
+    - containerPort: 6443
+      name: https
+      protocol: TCP
+    readinessProbe:
+      failureThreshold: 3
+      httpGet:
+        path: /healthz
+        port: 6443
+        scheme: HTTPS
+      initialDelaySeconds: 5
```

Comment:
which means it succeeds with 35s, but flakes with 35? Looks fragile.

Reply:
I checked the logs, the OAuth server was actually alive but never got the request, something else seems to be wrong here